server/forms.py

```diff
@@ -240,9 +292,57 @@ class CreateTaskForm(BaseForm):
                                    default=False)
 
 class UploadSubmissionForm(BaseForm):
-    upload_files = FileField('Submission Files', [FileRequired()])
-    flag_submission = BooleanField('Flag this submission for grading',
-                                   default=False)
+    upload_files = BackupUploadField('Submission Files', [FileRequired()])
```

Comment:
Yeah, I think having flagging be a separate action is better, especially since it affects other backups than just the one being submitted